### PR TITLE
fix(DragnDrop): re-enabled drag and drop

### LINKED ISSUE #7484: image drag and drop not working
# Bug Report

## Steps to reproduce

1. open a community channel
2. drag and drop an image

expected result: should be added as an attachment

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #7484

### What does the PR do
StatusChatInput re-enabled drag and drop images

### Affected areas
StatusChatInput

### Screenshot of functionality (including design for comparison)
https://user-images.githubusercontent.com/31625338/194359441-afd32d2b-d892-4dfb-a2d5-7152aa065b06.mov

### CODE REVIEW

**Review by caybro — COMMENTED** [review]
It should imho be possible to drop anywhere in the window, or at least in the chat area, WDYT? CC @John-44

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -1276,4 +1196,83 @@ Item {
             keycardPopup.item.open()
         }
     }
+
+    DropArea {
```
Comment (micieslak) [inline ui/app/mainui/AppMain.qml:1200]: @alexandraB99, please exclude it to a separate file. We need to reduce size of this file step by step :)

**Review by micieslak — CHANGES_REQUESTED** [review]

**Comment by alexandraB99** [comment]
> It should imho be possible to drop anywhere in the window, or at least in the chat area, WDYT? CC @John-44

@caybro it's possible to drop everywhere atm, please check

**Comment by John-44** [comment]
> > It should imho be possible to drop anywhere in the window, or at least in the chat area, WDYT? CC @John-44
> 
> @caybro it's possible to drop everywhere atm, please check

Yes dropping anywhere is what we want, at least for now.

File: ui/imports/utils/Global.qml [hunk 2]
```diff
@@ -10,6 +10,7 @@ Item {
 
     property var applicationWindow
     property var appMain
+    property var dragArea
```
Comment (caybro) [inline ui/imports/utils/Global.qml:13]: It's a `dropArea` :)

**Review by caybro — APPROVED** [review]
Looks good!

File: ui/app/mainui/AppMain.qml [hunk 3]
```diff
@@ -1276,4 +1197,22 @@ Item {
             keycardPopup.item.open()
         }
     }
+
+    DropAreaPanel {
```
Comment (noeliaSD) [inline ui/app/mainui/AppMain.qml:1201]: Should it also be possible on public chats?

https://user-images.githubusercontent.com/97019400/194829066-f045fa45-37f1-40d3-819a-350dfc1d180f.mov

Comment (iurimatias) [reply ui/app/mainui/AppMain.qml:1201]: No, this feature is not supported in public chats, only group chats, 1 on 1s and community channels

**Review by iurimatias — APPROVED** [review]

**Pull request merged**